docs/getting-started.md

```diff
@@ -53,50 +55,61 @@ $ git log --show-signature
 Build and install globally:
 ```
 $ npm rebuild
-$ npm install -g # link globally
+$ npm install --global
 ```
 
-If you're updating a repository it is necessary to run `npm rebuild` again
-if any dependencies with native addons have been updated.
+Note: If you're updating a repository it is necessary to run `npm rebuild`
+again if any dependencies with native addons have been updated.
 
-### Installing on Debian/Ubuntu
+### Installing via GPK
 
-Install the necessary dependencies in addition to Node.js:
+To support signature verification, you can use [`gpk`][gpk] to replace the
+use of `npm`.
+
+To install `bcoin` globally and to your path:
 ```
-apt-get install build-essential python
+$ gpk install --global https://github.com/bcoin-org/bcoin
 ```
 
-### Installing via Docker
-
-Check [bcoin-docker](https://github.com/bcoin-org/bcoin-docker)
+To install `bcoin` as a dependency, you can create a new
+`package.json` with:
+```
+$ gpk init
+```
 
-### Installing on Windows
+And then add `bcoin` with:
+```
+$ gpk install https://github.com/bcoin-org/bcoin
+```
+The latest tagged version will be added to `package.json` and bcoin
+will be installed.
 
-Install OpenSSL v1.0.2L 64-Bit:
+To run the build:
+```
+$ gpk rebuild
+```
```

Comment:
There is one other option we could add which is to install locally without integrating into a package or installing globally: git clone / gpk rebuild. This is how I'll be using it for development, for example.

Reply:
Yeah, much is that is covered in "Installing with Git...".

Reply:
Moved to that section.